tests/dummy/app/templates/demos/sl-modal.hbs

```diff
@@ -0,0 +1,179 @@
+    \{{sl-modal-header title="Custom Footer"}}
+
+    \{{#sl-modal-body}}
+         &lt;p&gt;Custom footer with multiple buttons&lt;/p&gt;
+    \{{/sl-modal-body}}
+
+    \{{#sl-modal-footer}}
+        &lt;p&gt;Modal with multiple buttons in the footer&lt;/p&gt;
+        &lt;button type=&quot;button&quot; class=&quot;btn btn-primary&quot;&gt;Save changes&lt;/button&gt;
+   \{{/sl-modal-footer}}
+\{{/sl-modal}}</pre>
+
+            {{#sl-modal name="customFooter"}}
+                {{sl-modal-header title="Custom Footer"}}
+
+                {{#sl-modal-body}}
+                    <p>Modal with multiple buttons in the footer</p>
+                {{/sl-modal-body}}
+
+                {{#sl-modal-footer}}
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                    <button type="button" class="btn btn-primary">Save changes</button>
+                {{/sl-modal-footer}}
+            {{/sl-modal}}
+        </div>
+        <div class="col-lg-6">
+            <h6>Click to Show Component</h6>
+            {{sl-button openModal="customFooter" label="Open Modal"}}
+        </div>
+    </div>
+</section>
+
+<hr>
+<section>
+    <h2>Triggering a modal to open using a sl-button</h2>
+    <p>The code below shows how to use a {{#link-to 'demos.sl-button'}}sl-button{{/link-to}} to trigger a modal with a name of "demo" to open, when clicked.</p>
+    <p>Pass in the name of the modal as the value to the <code>openModal</code> property.</p>
+    <pre> \{{sl-button openModal="demo" label="Open Modal"}}</pre>
+</section>
+
+<hr>
+<section>
+    <h2>Using sl-modal service to manage multiple modals</h2>
+    <p>Every modal with a unique name is registered on the sl-modal service, refer to the {{#link-to 'demos.services.sl-modal'}}sl-modal service page{{/link-to}} for documentation and sample code on how to use the service to lookup and manage modals across the app.</p>
+</section>
+<hr>
+<h3>sl-modal Properties</h3>
+{{#property-text name="afterHide" type="function"}}
+    Action to call after modal is shown.
+{{/property-text}}
+
+{{#property-text name="afterShow" type="function"}}
+    Action to call after modal is shown.
+{{/property-text}}
+
+{{#property-text name="backdrop" type="boolean" default="true"}}
+    Whether to show Bootstrap's backdrop.
+{{/property-text}}
+
+{{#property-text name="beforeHide" type="function"}}
+    Action to call after modal is shown.
+{{/property-text}}
+
+{{#property-text name="beforeShow" type="function"}}
+    Action to call after modal is shown.
+{{/property-text}}
+
+{{#property-text name="hide" type="function"}}
+    Hide modal.
+{{/property-text}}
+
+{{#property-text name="name" type="string"}}
+    Unique name used to identify modal. This name can be used to lookup the modal using the {{#link-to 'demos.services.sl-modal'}}modal service{{/link-to}}.
+{{/property-text}}
+
+{{#property-text name="show" type="function"}}
```

Comment:
The list of properties are meant to delineate the ones a developer can pass to the component to affect functionality.  This property is not one of those and should be removed from this list.